source/danger.d.ts

```diff
@@ -200,10 +198,32 @@ interface BitBucketServerJSONDSL {
   activities: BitBucketServerPRActivity[]
 }
 
+interface BitBucketServerAPIDSL {
+  /** Gets the contents of a file from a repo (defaults to yours) */
+  getFileContents(filePath: string, repoSlug?: string, refspec?: string): Promise<string>
+
+  /** Make a get call against the bitbucket server API */
+  get(path: string, headers: any, suppressErrors?: boolean): Promise<any>
+
+  /** Make a post call against the bitbucket server API */
+  post(path: string, headers: any, body: any, suppressErrors?: boolean): Promise<any>
+
+  /** Make a put call against the bitbucket server API */
+  put(path: string, headers: any, body: any): Promise<any>
+
+  /** Make a delete call against the bitbucket server API */
+  delete(path: string, headers: any, body: any): Promise<any>
+}
+
```

Comment:
@NMinhNguyen - this is currently the only "exposed" API in the sense of auto-complete, the entire class instance is there with its real extra functions, you can expand this if there were specific functions you needed